LiteNetLibSystemExample/core/GameManager.cs

```diff
@@ -37,6 +38,13 @@ public GameManager()
     // Called when the node enters the scene tree for the first time.
     public override void _Ready()
     {
+        //check for args to override server settings from command line args
+        var args = OS.GetCmdlineArgs();
+        if (args.Contains("--server"))
+            IsServer = true;
+        if (args.Contains("--client"))
+            IsServer = false;
+        
         Input.MouseMode = IsPlaying ? Input.MouseModeEnum.Captured : Input.MouseModeEnum.Visible;
```

Comment:
When running multiple instances, we usually need to reorganize the windows before we can play around with the game, so I think we should remove it.
```suggestion
```